docs/how-to/gpu-executor.md

```diff
@@ -1,486 +0,0 @@
-(gpu-executor)=
```

Comment:
same, I believe we had a discussion before and team core agreed this section is valuable